pages/_lifeEvent.vue

```diff
@@ -340,15 +340,11 @@ export default {
         this.lifeEventBenefits = _.sortBy(this.lifeEventBenefits, [this.sort])
       } else {
         const forceToBottom = 2048
-        const virtualCriteria = this.getVirtualCriteria()
         this.lifeEventBenefits.forEach((benefit) => {
           const matches = (benefit.matches = this.getTotalEligibleCriteria(benefit.eligibility))
-          const ineligible = this.getTotalIneligibleCriteria(benefit.eligibility) > 0
-          const virtualBenefitEligibility = (benefit.virtualBenefitEligibility = (benefit.eligibility || []).filter(
```

Comment:
Tangentially-related to the PR, is there a way to simplify this and make it more readable?  It hurts my brain.

Reply:
We could split up the `+ (ineligible ? forceToBottom: 0)` into another separate variable if that would help 👍🏼